docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -472,11 +472,22 @@ In the **Create custom cloud** wizard, proceed as follows:
    * Region
    * CIDR
 
-     Aiven needs CIDR for the `CIDR block of the VPC <https://docs.aws.amazon.com/vpc/latest/userguide/vpc-cidr-blocks.html>`_ that will be created in your AWS account.
+     The **CIDR** block defines the IP address range of the VPC that Aiven creates in your AWS account. Any Aiven service created in your Aiven organization will be placed in the VPC and will get an IP address within this address range.
```

Comment:
This applies only to the custom cloud in question. On the org level other deployment targets can be used as well.

```suggestion
     The **CIDR** block defines the IP address range of the VPC that Aiven creates in your AWS account. Any Aiven service created in the custom cloud will be placed in the VPC and will get an IP address within this address range.
```